### ISSUE TITLE
Fill Bill of Materials

### ISSUE THREAD

**Issue by stefanieschreiner** [amosproj/amos2022ws02-automotive-test-app#27]
## User story
1. As a stakeholder
2. I want to be able know which materials have been used in the app
3. So that I can get a better picture of how the app is set up.

## Acceptance criteria
* The sheet "Bill of Materials" in the planning documents file has been filled.
* The planning documents have been uploaded to the deliverables folder of sprint-02 in PDF-format.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
